automatminer/automl/keras_wrapper.py

```diff
@@ -0,0 +1,157 @@
+from __future__ import unicode_literals
+import keras.models
+import keras.legacy.layers
+import keras.regularizers
+import sklearn.metrics
+import sklearn.base
+import keras.constraints
+import keras.layers.noise
+import keras.optimizers
+import keras.callbacks
+from sklearn.datasets import load_boston
+from sklearn.model_selection import KFold, cross_val_score
+from keras.wrappers.scikit_learn import KerasRegressor, KerasClassifier
+import pandas as pd
+from gen_evolve_v2 import NeuralNetOptimizer
+import numpy as np
+import random
+
+_classifier_modes = {'classifier', 'classification', 'classify'}
```

Comment:
Don't copy and paste constants like this.

Either: 
1. Import them from the other module.
2. Move them to a common file and import them to both modules.